pom.xml

```diff
@@ -127,7 +127,7 @@
                 <filtering>true</filtering>
                 <directory>${basedir}/src/main/resources/</directory>
                 <includes>
-                    <include>*</include>
+                    <include>**/*</include>
```

Comment:
Actually, yea. This is the and pattern matching, it lines up with bash mostly, and in bash it's not recursive.